bin/update-readmes.js

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+
+const path = require( 'path' );
+const childProcess = require( 'child_process' );
+
+const packages = [
+	'e2e-test-utils',
+];
+
+let aggregatedExitCode = 0;
+packages.forEach( ( packageName ) => {
+	const args = [
+		`packages/${ packageName }/src/index.js`,
+		`--output packages/${ packageName }/README.md`,
+		'--to-token',
+	];
+	const pathToDocGen = path.join( __dirname, '..', 'node_modules', '.bin', 'docgen' );
+	const { status, stderr } = childProcess.spawnSync(
```

Comment:
We've a bad habit in our scripts of making them blocking/synchronous when they don't need to be. 

See #14295 for a proposed revision.